packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -114,6 +114,16 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
   }
   ul {
     list-style-type: "→ ";
+
+    & > li {
+      margin: 1.1rem 0;
+      line-height: 1.3;
```

Comment:
According to the design, the line-height is 24px, which equals `line-height: 1.5rem`.